### ISSUE TITLE
Collapse large option sets in refusal advice Q&A

### ISSUE THREAD

**Issue by garethrees** [mysociety/alaveteli#6017]
There are lots of exemptions!

![Screenshot 2020-12-10 at 15 03 23](https://user-images.githubusercontent.com/282788/101789205-e8fb0f00-3af8-11eb-9555-d624c2a89350.png)

I think we should:

* Order by most common in the YAML file
* Only show the top 6 by default
* Have a "Show more" link to reveal the remainder
* Always show all of them if one outside the top 6 is pre-selected

**Comment by wrightmartin** [comment]
> Always show all of them if one outside the top 6 is pre-selected

When one 'below the line' is selected by the user should we allow them to hide again and keep it pre-selected?

Also, if we're doing this we should definitely consider moving away from the three columns to a single column of options, there's plenty of evidence that a single scannable list will be easier to comprehend than the grid approach

**Comment by zarino** [comment]
+1 for no longer being in columns – that was just a temporary hack from me during the first prototypes. Worked ok when there were three or four options, but deffo not any more.

**Comment by garethrees** [comment]
> When one 'below the line' is selected by the user should we allow them to hide again and keep it pre-selected?

Yeah, it doesn't need to be that clever. I just think we want a default of not bombarding users with ~30 checkboxes. The [blue "Learning and assessment" section here](https://www.cardiff.ac.uk/study/postgraduate/taught/courses/course/advanced-chemistry-msc) is basically what I was thinking.

> +1 for no longer being in columns

I don't disagree, but there's some logic in the [form builder](https://github.com/mysociety/alaveteli/blob/develop/app/helpers/refusal_advice_question_form.rb) that would need to change, and we don't have spare programmer capacity for all the things we're doing right now.

I don't want us to get held up on this, so let's create a separate issue for this suggestion. The "too many options" problem needs solving irrespective of whether we use a column or grid, so let's solve that first.

**Comment by garethrees** [comment]
> changes in whatdotheyknow-theme or alaveteli?
>
> – @wrightmartin (slack)


Alaveteli plz! It should “Just work” for any theme that renders the refusal advice.

**Comment by garethrees** [comment]
> Order by most common in the YAML file

Here's a sorted version of the top exemptions identified on the 10,000 most recent requests (Oct 2020) https://github.com/mysociety/alaveteli/issues/5912#issuecomment-717186502

```ruby
 {"Section 12"=>371,
 "Section 1"=>297,
 "Section 14"=>288,
 "Section 17"=>279,
 "Section 40"=>266,
 "Section 21"=>169,
 "Section 31"=>137,
 "Section 16"=>127,
 "Section 43"=>110,
 "Section 50"=>96,
 "Section 8"=>95,
 "Section 10"=>85,
 "Section 30"=>84,
 "Section 38"=>78,
 "Section 41"=>60,
 "Section 2"=>60,
 "Section 22"=>53,
 "Section 35"=>49,
 "Section 44"=>44,
 "Section 36"=>38,
 "Section 5"=>34}
```
